### PR TITLE
Start capturing changes in log

### PR DESCRIPTION
**Pull request by Twon**

### CODE REVIEW

File: DRAFT.md [hunk 1]
```diff
@@ -40,6 +40,80 @@ This proposal is a fusion of two earlier individual proposals, P1950 and P0201.
...
+
+* Add `std::format` support for `std::indirect`
+
+* Add Appendix B: Before and after examples.
+
+* Add preconditions checking for types are not valueless.
+
+* Add constexpr support.
+
+* Allow quality of implentation support for small buffer optimization for Polymophic.
```
Comment (jbcoe) [inline DRAFT.md:103]: Typo implentation -> implementation

File: DRAFT.md [hunk 2]
```diff
@@ -40,6 +40,80 @@ This proposal is a fusion of two earlier individual proposals, P1950 and P0201.
...
+
+* Ensure swap and assign with self are nops.
+
+* Move feature test macros to [version.syn].
+
+* Remove `std::optional` specializations.
+
+* Replace use of "erroneous" with "a programming error".
+
+* Strong exception guarantee for copy assignement.
```
Comment (jbcoe) [inline DRAFT.md:77]: Typo assignement -> assignment

File: DRAFT.md [hunk 3]
```diff
@@ -40,6 +40,80 @@ This proposal is a fusion of two earlier individual proposals, P1950 and P0201.
...
+
+* Address constraints wording for `std::indirect` comparison operators.
+
+* Copy constructor now uses allocator_traits select_on_container_copy_construction.
+
+* Ensure swap and assign with self are nops.
+
+* Move feature test macros to [version.syn].
+
+* Remove `std::optional` specializations.
```
Comment (jbcoe) [inline DRAFT.md:73]: I think we did this in an earlier revision, happy to be wrong.

**Review by jbcoe — APPROVED** [review]

File: DRAFT.md (hunk 3 above)
Comment (Twon) [reply DRAFT.md:73]: The change we did this in was: https://github.com/jbcoe/value_types/commit/7fec8a37a74ff50bc0456041402565a6296a942e

This was on November 11th (the Saturday we made many changes in the hotel lobby), we had previously presented R2 of the paper at the end of Friday the afternoon before. So, I'm pretty sure its this revision unless I've missed something else?

Comment (jbcoe) [reply DRAFT.md:73]: My mistake. Thanks for double checking.

**Review by jbcoe — APPROVED** [review]

**Pull request merged**